services/frontend-service/pkg/handler/handle_test.go

```diff
@@ -833,29 +864,29 @@ func (m *mockDeployClient) ReleaseTrain(_ context.Context, in *api.ReleaseTrainR
 	return &api.ReleaseTrainResponse{Target: in.Target, Team: in.Team}, nil
 }
 
-type mockLockClient struct {
-	createEnvironmentLockRequest            *api.CreateEnvironmentLockRequest
-	deleteEnvironmentLockRequest            *api.DeleteEnvironmentLockRequest
-	createEnvironmentApplicationLockRequest *api.CreateEnvironmentApplicationLockRequest
-	deleteEnvironmentApplicationLockRequest *api.DeleteEnvironmentApplicationLockRequest
+type mockBatchClient struct {
+	batchRequest *api.BatchRequest
+	//deleteEnvironmentLockRequest            *api.DeleteEnvironmentLockRequest
```

Comment:
Why just commented out? Git has history! Just remove it.

Reply:
done